posts/2021-03-24-whats-new-in-MP-Rest-Client2.0.adoc

```diff
@@ -22,9 +22,10 @@ take a look at some of the new features and how you can use them in Open Liberty
 
 == Getting started
 
-The MicroProfile Rest Client 2.0 implementation is new in Open Liberty 21.0.0.3 - so make sure you are using the latest
-version of Liberty. Next, you will need to add the following link:{url-prefix}/guides/maven-intro.html[Maven] dependency
-to your pom.xml:
+The MicroProfile Rest Client 2.0 implementation is new in 
```

Comment:
Can you add something that mentions that it's part of the larger MP 4.0 release, and link to https://openliberty.io/blog/2021/03/19/microprofile40-open-liberty-21003.html